File: urionlinejudge/1010 - Simple Calculate.java
// Created by islam elhady on 1/21/18.

import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
 
    
          Scanner s = new Scanner (System.in);
              
              int A , B ; ;
              float C , value ;
              
              A = s.nextInt();
              B = s.nextInt();
              C = s.nextFloat();
              
              value = B * C ;
              
              A = s.nextInt();
              B = s.nextInt();
              C = s.nextFloat();
              
              value += B * C ;
              System.out.printf("VALOR A PAGAR: R$ %.2f\n", value);
 
    }
 
}

File: urionlinejudge/1010 - Simple Calculate.cpp
// Created by islam elhady on 9/5/20.

#include <iostream>
 
using namespace std;
 
int main() {
 
    int x1, y1, x2, y2;
  double z1, z2;

  scanf("%d %d %lf %d %d %lf", &x1, &y1, &z1, &x2, &y2, &z2);

  printf("%s: R$ %.2f\n", "VALOR A PAGAR", (y1 * z1) + (y2 * z2));

  return 0;
}